src/Neo/Wallets/NEP6/NEP6Wallet.cs

```diff
@@ -86,13 +93,20 @@ public NEP6Wallet(string path, string password, ProtocolSettings settings, JObje
 
         private void LoadFromJson(JObject wallet, out ScryptParameters scrypt, out Dictionary<UInt160, NEP6Account> accounts, out JToken extra)
         {
-            version = Version.Parse(wallet["version"].AsString());
-            name = wallet["name"]?.AsString();
-            scrypt = ScryptParameters.FromJson((JObject)wallet["scrypt"]);
-            accounts = ((JArray)wallet["accounts"]).Select(p => NEP6Account.FromJson((JObject)p, this)).ToDictionary(p => p.ScriptHash);
-            extra = wallet["extra"];
-            if (!VerifyPasswordInternal(password))
-                throw new InvalidOperationException("Wrong password.");
+            try
+            {
+                version = Version.Parse(wallet["version"].AsString());
+                name = wallet["name"]?.AsString();
+                scrypt = ScryptParameters.FromJson((JObject)wallet["scrypt"]);
+                accounts = ((JArray)wallet["accounts"]).Select(p => NEP6Account.FromJson((JObject)p, this)).ToDictionary(p => p.ScriptHash);
+                extra = wallet["extra"];
+                if (!VerifyPasswordInternal(password))
+                    throw new WalletException(WalletErrorType.PasswordIncorrect, "Wrong password.");
```

Comment:
some exceptions ends with '.' and other don't

Reply:
will check and fix.

Reply:
fixed